modules/openy_features/openy_node/modules/openy_node_social_post/config/install/views.view.social_posts.yml

```diff
@@ -0,0 +1,597 @@
+          element_label_colon: true
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          destination: true
+          entity_type: node
+          plugin_id: entity_operations
+      filters:
+        status:
+          value: '1'
+          table: node_field_data
+          field: status
+          plugin_id: boolean
+          entity_type: node
+          entity_field: status
+          id: status
+          expose:
+            operator: ''
+          group: 1
+        type:
+          id: type
+          table: node_field_data
+          field: type
+          value:
+            social_post: social_post
+          entity_type: node
+          entity_field: type
+          plugin_id: bundle
+        field_platform_value:
+          id: field_platform_value
+          table: node__field_platform
+          field: field_platform_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          operator: '='
+          value: ''
+          group: 1
+          exposed: true
+          expose:
+            operator_id: field_platform_value_op
+            label: 'Platform (field_platform)'
+            description: null
+            use_operator: false
+            operator: field_platform_value_op
+            identifier: field_platform_value
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
+              authenticated: authenticated
+          is_grouped: true
+          group_info:
+            label: Platform
+            description: ''
+            identifier: field_platform_value
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items:
+              1:
+                title: Facebook
+                operator: '='
+                value: Facebook
+              2:
+                title: Twitter
+                operator: '='
+                value: Twitter
+              3:
+                title: Instagram
+                operator: '='
+                value: Instagram
+          plugin_id: string
+      sorts:
+        field_posted_value:
+          id: field_posted_value
+          table: node__field_posted
+          field: field_posted_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          order: DESC
+          exposed: false
+          expose:
+            label: ''
+          granularity: second
+          plugin_id: datetime
+      title: 'Social Posts'
+      header: {  }
+      footer: {  }
+      empty: {  }
+      relationships: {  }
+      arguments: {  }
+      display_extenders: {  }
+    cache_metadata:
+      max-age: 0
```

Comment:
can we set cached bu time and add 1h at least?
This view will be displayed on multiple pages, we should be careful about disabling cache, because it'll block whole page caching

Reply:
1h added